docs/learn/learn-async-backing.md

```diff
@@ -7,103 +7,232 @@ keywords: [parachains, backing, parablock, perspective parachains, unincluded se
+Unincluded segments are chains of candidate parablocks that are seconded but still need to be backed
+and included in the relay chain. Parablocks can be added to this unincluded segment without waiting
+for the latest included parent block of the relay chain. The core functionality that asynchronous
+backing brings is the ability to build on these unincluded segments of block ancestors rather than
+ancestors included in the relay chain state.
 
-1. The parablock is not placed on the relay chain in its entirety, but rather the **paraheader**
-2. The relay chain does not access the entire state of a parachain but only the values that changed
-   during that block and the merkelized hashes of the unchanged values.
+:::info Unincluded segments happen on the collator portion
+
+**Unincluded** segments are blocks that are seconded but still need to be backed. The backing
+process occurs on the relay chain, whereas unincluded segments are moreso held in the collator.
 
 :::
 
-For more information on the validity and availability process, be sure to visit the
-[parachain protocol](../learn/learn-parachains-protocol.md) page.
+Compared to synchronous backing, contextual execution shifts from being the parablock ancestors
+included in the relay chain to being the latest ancestor parablock pushed into the unincluded
+segment. This allows collators to build parablocks earlier, giving them plenty of time to fit more
+transactions and prepare block candidates for backing and inclusion.
+
+### Asynchronous Backing Mechanics
+
+In asynchronous backing, parablock generation is decoupled from backing and inclusion. Compared to
+[synchronous backing](#synchronous-backing-mechanics):
+
+- Parablocks are not rushing to being backed, they are not placed on the conveyor belt when they are
+  generated
+- Filled blocks are placed in the unincluded segment, and from there, they are injected onto the
+  conveyor belt
+- The conveyor belt can carry multiple parablocks ([pipelining](#pipelining)), and a parablock can
+  be in a different stage from another one being built if it abides by the parameters set forth by
+  the asynchronous backing configuration.
+- Backing and inclusion events can happen within the same relay chain block (i.e. in 6 seconds)
+- The last parablock in the unincluded segment is used as execution context to generate a new
+  parablock and, because the unincluded segment can carry multiple parablocks, parachains can take
+  more time to fill new blocks while the segment is emptied.
+
+The diagram below shows parablocks on their way from being generated to being backed and included
+into the relay chain in the context of asynchronous backing.
+
+![async-backing](../assets/async-backing.png)
+![async-backing-legend](../assets/async-backing-legend.png)
+
+Parablock 1 (P1) is included, and P2 is backed within R1. In the meantime, the unincluded segment is
+full and contains P3-5. Because P6 is not rushing to be backed, it can be filled in 6s while P2 is
```

Comment:
Even with async backing, we're only recommending 2 seconds of block building/filling time. This limit may change slightly in the future as we optimize block validation in the backing and approval steps.